package.json

```diff
@@ -2,7 +2,8 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.29.0",
+  "version": "0.30.0",
+  "main": "dist/index.js",
```

Comment:
Would this now be `fozzie.js`?

Reply:
I thought that too, but because that's the main entry point (`index.js`) it shouldn't need updating. :)

Reply:
@xander-marjoram you're going to need to update the [`prepare` npm script](https://github.com/justeat/fozzie/blob/418ace8f26e8fba20cc7d823ce2f533eab53618e/package.json#L60) so that the JS is compiled and output into the `dist` folder. 

We use the [`concurrently` npm module](https://github.com/justeat/f-header/blob/20ac75eb309f373c2a8886f974a98d40168abbe9/package.json#L56) to run multiple tasks in parallel.

Reply:
@kevinrodrigues the file which is output into the `dist` directory is called `fozzie.js` now though so the JS wouldn't be accessible?

Reply:
Have just seen your other comment below @kevinrodrigues, so this line should be fine once the `prepare` script has been updated 🍃